launch/state_publisher.launch

```diff
@@ -7,12 +7,4 @@
     <node pkg="robot_state_publisher" type="robot_state_publisher" name="state_publisher"/>
 
     <node pkg="herb_launch" type="segway_joint_relay.py" name="relay" ns="segway"/>
-
-    <node pkg="herb_launch" type="owd_joint_relay.py" name="relay" ns="head/owd"/>
-
-    <node pkg="herb_launch" type="owd_joint_relay.py" name="relay" ns="right/owd"/>
-    <node pkg="herb_launch" type="bhd_joint_relay.py" name="relay" ns="right/bhd"/>
-
-    <node pkg="herb_launch" type="owd_joint_relay.py" name="relay" ns="left/owd"/>
-    <node pkg="herb_launch" type="bhd_joint_relay.py" name="relay" ns="left/bhd"/>
 </launch>
```

Comment:
You should also delete the `left_arm.launch`, `right_arm.launch`, and `head.launch` if we are completely removing functionality related to OWD.